internal/query/projection/failed_events.go

```diff
@@ -0,0 +1,8 @@
+package projection
+
+const (
+	FailedEventsColumnProjectionName = "projection_name"
```

Comment:
Im not sure if we should create the consts here because we don't need them here

Reply:
moved it to query pkg and made it private